test/build/50_ch-image.bats

```diff
@@ -31,6 +31,31 @@ setup () {
     [[ $output = *'verbose level: 1'* ]]
 }
 
+@test 'ch-image delete' {
+    # delete/test image doesn't exist
+    run ch-image list
+    echo "$output"
+    [[ $status -eq 0 ]]
+    [[ $output != *"delete/test"* ]]
+  
+    # builds image 
+    # called delete/test to check name parsing
+	 ch-image build -t delete/test -f - . << 'EOF'
+FROM 00_tiny
+EOF
+    run ch-image list
+	 echo "$output"
+	 [[ $status -eq 0 ]]
+	 [[ $output = *"delete/test"* ]]
```

Comment:
Looks like some [tab characters](https://www.jwz.org/doc/tabs-vs-spaces.html) snuck in (project style is spaces only):

```suggestion
    run ch-image list
    echo "$output"
    [[ $status -eq 0 ]]
    [[ $output = *"delete/test"* ]]
```